src/source/benchmarks/ArrayAccessDirectVsNode.bs

```diff
@@ -0,0 +1,32 @@
+@suite()
+namespace ArrayAccessDirectVsNode
+
+  sub setup()
+    size = 1
```

Comment:
Should this number be larger?

Reply:
my original intent was to use it to crank up the number of iterations, but I quickly blew out channel memory so I left it small